features/support/commit_helpers.rb

```diff
@@ -155,7 +161,11 @@ def recent_commit_shas count
 
 # Returns the SHA of the commit with the given message
 def sha commit_message
-  output_of "git reflog --grep-reflog='commit: #{commit_message.strip}' --format='%H'"
+  if commit_message == 'Initial commit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.